MagneticFieldofStraightwire.md

```diff
@@ -0,0 +1,53 @@
+# Documentation
+
+## Magnetic Field of Straight wire
+
+It is understood that a moving charge or current generates a magnetic field. A long straight wire carrying a current I is 
+shown in diagram below. The presence of current in the wire creates a magnetic field around it which consist of concentric
+circles. 
+![Screenshot 2024-10-13 164423](https://github.com/user-attachments/assets/25840c22-e22b-402f-86fb-426b548127b3)
```

Comment:
If you do not have the copyright to this image please remove. If you have it needs a reference.

Reply:
Hello, I have made this picture by myself.